### ISSUE TITLE
What do we do with the jetstack-dev GCP project?

### ISSUE THREAD

**Issue by maelvls** [jetstack/jetstack-secure-gcm#25]
We created [jetstack-public](https://console.cloud.google.com/home/dashboard?project=jetstack-public) and [jetstack-dev](https://console.cloud.google.com/home/dashboard?project=jetstack-dev) for the Marketplace purposes, but we only seem to be using [jetstack-public](https://console.cloud.google.com/home/dashboard?project=jetstack-public).

Should we decommission [jetstack-dev](https://console.cloud.google.com/home/dashboard?project=jetstack-dev)? It has nothing running on it at the moment

**Comment by maelvls** [comment]
@simplyzee Do you know if it is OK to remove the `jetstack-dev` project since we don't need it for the Google Cloud Marketplace?

**Comment by simplyzee** [comment]
@james-w i remember the dev project being a requirement at the time when you requested it. Do you think its still needed now?

**Comment by james-w** [comment]
Yeah, they said that it was mandatory for us to create it, but I don't know why, and we haven't
used it.

We could try deleting it and see if they complain, and reinstate it if they do.

**Comment by maelvls** [comment]
@chance19 How can we get the project `jetstack-dev` deleted?
